Jenkinsfile

```diff
@@ -936,22 +943,45 @@ pipeline{
                     }}}
             }
 			post{
-				always{
-				  	sh 'rm -rf $WORKSPACE/xdmp'
-				  }
-                  success {
+             success {
                     println("End-End Tests Completed")
                     sendMail Email,'<h3>Tests Passed on Released 9.0 ML Server Single Node </h3><h4><a href=${RUN_DISPLAY_URL}>Check the Pipeline View</a></h4><h4> <a href=${BUILD_URL}/console> Check Console Output Here</a></h4>',false,'$BRANCH_NAME branch | Linux RH7 | ML-9.0-11 | Single Node | Passed'
 
-                   }
-                   unstable {
+             }
+             unstable {
                       println("End-End Tests Failed")
                       sh 'mkdir -p MLLogs;cp -r /var/opt/MarkLogic/Logs/* $WORKSPACE/MLLogs/'
                       archiveArtifacts artifacts: 'MLLogs/**/*'
                       sendMail Email,'<h3>Some Tests Failed on Released 9.0 ML Server Single Node </h3><h4><a href=${JENKINS_URL}/blue/organizations/jenkins/Datahub_CI/detail/$JOB_BASE_NAME/$BUILD_ID/tests><font color=red>Check the Test Report</font></a></h4><h4><a href=${RUN_DISPLAY_URL}>Check the Pipeline View</a></h4><h4> <a href=${BUILD_URL}/console> Check Console Output Here</a></h4><h4>Please create bugs for the failed regressions and fix them</h4>',false,'$BRANCH_NAME branch | Linux RH7 | ML-9.0-11 | Single Node | Failed'
-                  }
-                  }
+             }}
         }
+        stage('fullCycle-rh7-singlenode-9.0-11') {
+        agent { label 'dhfLinuxAgent' }
+        environment{
+                    JAVA_HOME="$JAVA_HOME_DIR"
+                    PATH="$JAVA_HOME:$PATH"
+                }
+        steps {timeout(time: 3, unit: 'HOURS') {
+                  catchError(buildResult: 'SUCCESS', catchInterruptions: true, stageResult: 'FAILURE') {
+                            cleanWs()
+                            copyRPM 'Release', '9.0-11'
+                            setUpML '$WORKSPACE/xdmp/src/Mark*.rpm'
+                            sh 'cd $WORKSPACE/data-hub;./gradlew -g ./cache-build clean ml-data-hub:testFullCycle -i --stacktrace'
```

Comment:
Why do we need another stage for running full cycle tests? Can't we. just run after running all the tests in the same stage?

Reply:
1) new stage would make executable time faster and better monitoring.
2)FullCycle is separate test and technically requires  different environment